utils.groovy

```diff
@@ -97,8 +97,7 @@ def bump_builds_json(stream, buildid, arch, s3_stream_dir) {
     lock(resource: "bump-builds-json-${stream}") {
         def remotejson = "s3://${s3_stream_dir}/builds/builds.json"
         aws_s3_cp_allow_noent(remotejson, './remote-builds.json')
-        shwrap("""
-        export AWS_CONFIG_FILE=\${AWS_BUILD_UPLOAD_CONFIG}
+        shwrapWithAWSBuildUploadCredentials("""
```

Comment:
Ahh, there it is. :) This needs to be in the other PR that adds that function instead.

Reply:
will move to the other PR.